buildchow/todo.adoc

```diff
@@ -0,0 +1,69 @@
+= Advantages noticed within a week:
+
+* fixing JRE across all jobs
+* relasing target platform without breaking builds
+* using 9.0 instead of 4.3.mars for stream
+
+= Local Jenkins Configuration
+
+Things I needed to do to run/test on local Jenkins (not complete yet!)
+
+. Add label to have it be recognized as a runnable node. (i.e. RHEL7)
+. Add jdk jdk1.7 and jdk1.6 to have a server to look up.
+. Add maven installers
+. enable safe html under security
+. Add native_tools / sep (todo)
+. Install the plugins! (looking into using something like https://github.com/maxandersen/jenkins-ootb)
+ 
+= Todos
+
+Open:
+. buildpublish to jboss.org does not require name but publisher jenkins does (? new deploy might make this invalid)
+
+Done:
+. support downstream-ext https://wiki.jenkins-ci.org/display/JENKINS/Downstream-Ext+Plugin (done: https://review.openstack.org/#/c/158117/)
+. be able to turn builder on/off (done: `disabled: true|false`)
+. handle JAVA17 property so its not a variable. (done: not used anymore in original jenkins)
+. do tests timeout (done: just using normal build timeout)
+. check if pattern are correct. (done: using variable to handle difference of the names)
+. missing AbortOperation for timeout (done: Seems abort is the default so not needed to mimic generated config.xml)
+. missing xvnc (done: https://review.openstack.org/#/c/156086/)
+. update via internal server result in cert error (done: https://review.openstack.org/#/c/178794/)
+. add way to not having to add direct support to jjb for a plugin (done: https://review.openstack.org/#/c/160158/)
+
+= Questions
+
+Some of these have been answered - just kept here until verified nothing missed yet.
+
+why no email notificiaton on version watch ? 
+
+why no triggers on version watch ? 
+
+why does versionwatch job define the include/excludes and not in our git repo so versioned/tracable togheter with rest of config ?
+
+why does product ignore failures and errors in mvn build step ?
+
+what does "targetPlatformGroup=jbdevstudio" mean ?
+
+why does product not build its updatesites in mvn instead of separate shell ?
+
```

Comment:
Product only has one update site. And it's part of the main mvn reactor. Not sure what you mean here. Are you talking about the step that COPIES the update site (which contains no packed jars) on top of the one that does, so that the zip is smaller, but the unpacked site DOES contain pack.gz artifacts?